Fix service now v2 escape (#15652)
demisto/content: Packs/ServiceNow/ReleaseNotes/2_2_12.md

#### Integrations
##### ServiceNow v2

> - When querying a group with an ampersand (&) the query would break. Added a fix via a code flag to not parse ampersands.

Reviewer: ```suggestion
- Fixed an issue where the ***servicenow-query-groups*** command failed when the group name contained an ampersand (&).
```
```suggestion
- When querying a group with an ampersand (&) the query would break. Added a fix via a code flag to not parse ampersands.
```

#### Integrations

> 

Reviewer: ```suggestion
```

##### ServiceNow v2

> 

Reviewer: ```suggestion
```
